core-services/prow/02_config/generate-boskos.py

```diff
@@ -47,7 +47,13 @@
         'us-east-1': 8,
         'us-east-2': 8,
         'us-west-2': 8
-    }
+    },
+    'aws-2-quota-slice': {
```

Comment:
nit: can we move this up next to `aws-quota-slice`?  Because it seems like the same group of folks would be looking at both.